buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -31,18 +25,18 @@
   {%- elif type == "double" -%}
     doubleKey
   {%- else -%}
-    {{lowerFirst(type)}}Key
+    {{type | to_camelcase(False)}}Key
   {%- endif -%}
 {%- endmacro %}
 {%- macro print_value(type, value) -%}
```

Comment:
nit: you can move it and use built-in 

``{{ attribute | print_member_value(member) }}``

instead.

see https://github.com/open-telemetry/opentelemetry-python/blob/cbebd89c6da683c0516665a929774062ae93b42d/scripts/semconv/templates/semantic_attributes.j2#L50 for full example